migration-engine/connectors/sql-migration-connector/src/flavour.rs

```diff
@@ -65,6 +65,77 @@ pub(crate) trait SqlFlavour:
     ) -> SqlResult<SqlSchema>;
 }
 
+#[derive(Debug)]
+pub(crate) struct MssqlFlavour(MssqlUrl);
+
+impl MssqlFlavour {
+    pub(crate) fn schema_name(&self) -> &str {
```

Comment:
Note: we might not need this.

Reply:
Yes, we have to clean that up at some point.

Reply:
We do need this...